### PR TITLE
{AKS} Improve Windows OutboundNat test case by removing Windows OSSKU limitation

### PR DESCRIPTION
**Pull request by ShiqianTao**
{AKS} Improve Windows OutboundNat test case by removing Windows OSSKU limitation

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command
```
az aks nodepool add --name npwin --os-type Windows --disable-windows-outbound-nat
```

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [x] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

**Review by AbelHu — APPROVED** [review]
LGTM. Thanks for the PR

**Comment by AbelHu** [comment]
Please investigate why the tests failed.

**Comment by yonzhan** [comment]
Please fix CI issues

**Comment by FumingZhang** [comment]
Failed CI caused by dependency bump ([#28847](https://github.com/Azure/azure-cli/pull/28847)), will fix in another PR.

**Review by FumingZhang — APPROVED** [review]
LGTM

File: src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py [hunk 1]
```diff
@@ -2798,16 +2798,14 @@ def test_aks_nodepool_add_with_disable_windows_outbound_nat(
             ],
         )
 
-        # add Windows2019 nodepool
-        # Windows2022 does not support this feature yet
+        # add Windows nodepool with disable windows outbound NAT
```
Comment (FumingZhang) [inline src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py:2801]: Author queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=93064731&view=results), test passed!
- test_aks_nodepool_add_with_disable_windows_outbound_nat

**Comment by ShiqianTao** [comment]
Hi @zhoxing-ms and @yanzhudd Could you please help to review/merge this PR? Thank you.

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**